src/app/overview/page.module.css

```
@@ -0,0 +1,83 @@
+.mainContainer > * {
+  margin-bottom: var(--space-5xl);
+}
+
+.headLine {
+  margin-top: var(--space-2xl);
+}
+
+.firstGroup {
+  padding-left: var(--space-md);
+  padding-right: var(--space-md);
+  padding-bottom: var(--space-md);
+}
+
+.noPadding {
+  padding: 0;
+}
+
+.firstGroup > * {
+  margin-bottom: var(--space-xl);
+}
+
+.firstGroup > *:last-child {
+  margin-bottom: 0;
+}
+
+.secondGroup {
+  padding-left: var(--space-md);
+  padding-right: var(--space-md);
+  padding-bottom: var(--space-md);
+}
```

`firstGroup`も`secondGroup`も似たようなスタイルなので共通化できそうです
paddingはtopを除く必要はないと思います
```
.firstGroup > * {
  margin-bottom: var(--space-xl);
}

.firstGroup > *:last-child {
  margin-bottom: 0;
}
```
の部分は
```
.section > * + * {
  margin-top: var(--space-xl);
}
```
とするとキレイです
```suggestion
.section {
  padding: var(--space-md);
}

.section > * + * {
  margin-top: var(--space-xl);
}
```